### DISCUSSION TITLE
I've made a Tutorial for LuaSnip

### DISCUSSION THREAD

**Discussion by ziontee113** [L3MON4D3/LuaSnip#405] [Show and tell]
So I just uploaded my LuaSnip Tutorial for those who need.

**https://youtu.be/ub0REXjhpmk**

![thumbnail 2-4](https://user-images.githubusercontent.com/102876811/165839973-a92d5a9f-732f-4460-9cec-8096ac1d04c8.jpg)

**Comment by L3MON4D3** [comment]
Oh hey, that's awesome :heart:
I really like that your video shows what is possible and how to get there, nicely complements the current Documentation :D (which is much too in-depth to get an overview of quickly :sweat_smile:)

**Reply by ziontee113** [reply]
Yeah. A casual person just won't get it. No matter how many cool examples you throw at them. Snippets, especially LuaSnip is too subtle they just overlook it.

**Reply by L3MON4D3** [reply]
Complex, too :sweat_smile:
But Videos like yours go a long way in pointing newcomers into the right directions, and the benefits of eg. luasnip are much more obvious (show, don't tell and all that :D ) :+1:

**Reply by ziontee113** [reply]
Yeah I'm planning to make more. But I need to get better first XD
